pkg/core/helper_test.go

```diff
@@ -428,8 +428,10 @@ func addNetworkFee(bc *Blockchain, tx *transaction.Transaction, sender *wallet.A
 	return nil
 }
 
+// Signer can be either bool or *wallet.Account.
+// In the first case `true` means sign by committee, `false` means sign by validators.
 func prepareContractMethodInvokeGeneric(chain *Blockchain, sysfee int64,
-	hash util.Uint160, method string, isCommittee bool, args ...interface{}) (*transaction.Transaction, error) {
+	hash util.Uint160, method string, signer interface{}, args ...interface{}) (*transaction.Transaction, error) {
```

Comment:
It's more like internal test function with explicit fail in case we do something bad. May be postpone this to https://github.com/nspcc-dev/neo-go/issues/1677 ?